packages/copyable/README.md

```diff
@@ -0,0 +1,85 @@
+# Copyable
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/copyable.svg)
+
+#### [View on Storybook](https://mongodb.github.io/leafygreen-ui/?path=/story/copyable--default)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/copyable
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/copyable
+```
+
+## Example
+
+```tsx
```

Comment:
can we keep this as `js` for now. Need this to be standardized across readmes for .design